becquerel/core/fitting.py

```diff
@@ -892,7 +893,11 @@ def _calc_area(param_vec, **kwargs):
         else:
             covariance = np.array(self.result.covar)
         if not covariance.sum():
```

Comment:
Why not use `if np.isclose(covariance.sum(), 0):`?

Reply:
Or is it returning `None`?

Reply:
Yes, it was returning None, and then np.array(None).sum() was behaving strangely. Cleaned up in the latest commit!